.dependebot/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 1
```

Comment:
typo: should be `.dependabot`

Reply:
https://github.com/jenkinsci/plugin-compat-tester/pull/182